packages/amplify-cli-core/API.md

```diff
@@ -779,6 +779,7 @@ export const executeHooks: (hooksMetadata: HooksMeta) => Promise<void>;
 
 declare namespace ExeInfo {
     export {
+        ExeInfo_2 as ExeInfo,
```

Comment:
This is going to result in a type `ExeInfo.ExeInfo`. Try finding a more precise name for `ExeInfo.ExeInfo`.

Reply:
I need to look into this. I had some code that I had created for a test, but it should have been deleted.

Reply:
Was created as a way to easily type exeInfo but was not needed. Deleted for now - we can always re-add if we discover a use for it.